src/main/java/com/gmail/nossr50/locale/LocaleLoader.java

```diff
@@ -85,7 +91,12 @@ else if (myLocale.length >= 2) {
                 locale = new Locale(myLocale[0], myLocale[1]);
             }
 
-            bundle = ResourceBundle.getBundle(BUNDLE_ROOT, locale);
+            Path localePath = Paths.get(mcMMO.getMainDirectory(), "locale_" + (locale != null ? locale.toString() : "") + ".properties");
```

Comment:
`mcMMO.getMainDirectory()` has slash in the end, `Paths.get(...)` does not like that really.

Reply:
Also not sure how to handle `locale == null` right now, I need to look how `ResourceBundle.getBundle(...)` works and then skip attempting to load the locale from filesystem at all

Reply:
Since `mcMMO.getMainDirectory()` returns a path with slash in the end, I simply proceeded with string concatenation.

As `ResourceBundle.getBundle(...)` throws `NullPointerException` when `locale` parameter is `null`, then I added a check to ensure that `locale` is not null.